app/api/images/images.list.controller.js

```diff
@@ -557,3 +563,131 @@ exports.getStats = utils.route(async (req, res) => {
     rows: result
   });
 });
+
+exports.getImagesBound = utils.route(async (req, res) => {
+  const query = req.query;
+  const attributes = parseAttributes(query);
+  const orderBy = query.sortKey;
+  const orderByNearest = models.sequelize.literal(`images.location <-> st_setsrid(ST_makepoint(${query.longitude}, ${query.latitude}), 4326)`);
+
+  let whereClauses = [];
+
+  if (query.owner_id) {
+    whereClauses.push({ owner_id: inUniqueOrList(query.owner_id) });
+  }
+
+  if (query.collection_id) {
+    whereClauses.push({ collection_id: inUniqueOrList(query.collection_id) });
+  }
+
+  if (query.keyword) {
+    whereClauses.push({
+      [Op.or]: {
+        original_id: iLikeFormatter(query.keyword),
+        title: iLikeFormatter(query.keyword),
+        caption: iLikeFormatter(query.keyword)
+      }
+    });
+  }
+
+  if (query.place_names) {
+    whereClauses.push({ geotags_array: containsUniqueOrList(query.place_names) });
+  }
```

Comment:
geotags_array in image is the old place to find the toponyms, the new place is in geometadata table.

I think it's better to use the new place.

Reply:
Not actually used on Smapshot frontend but we can take note to update this.